avm/res/network/dnsresolver/tests/e2e/defaults/main.test.bicep

```diff
@@ -54,18 +54,14 @@ module testDeployment '../../../main.bicep' = [for iteration in [ 'init', 'idem'
   name: '${uniqueString(deployment().name, location)}-test-${serviceShort}-${iteration}'
   params: {
     name: '${namePrefix}${serviceShort}001'
-    virtualNetworkId: nestedDependencies.outputs.virtualNetworkId
-    tags: {
-      'hidden-title': 'This is visible in the resource name'
-      Environment: 'Non-Prod'
-      Role: 'DeploymentValidation'
-    }
+    virtualNetworkResourceId: nestedDependencies.outputs.virtualNetworkId
```

Comment:
Should ideally also reference a `resourceId`. I believe it's a good practice to be explicit by example

Reply:
The same applies to the other test files

Reply:
see previous comments :)